schemas/common/b2b-contact.schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-contact",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Contact",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Contact specific attributes.",
+  "definitions": {
+    "b2b-contact": {
+      "properties": {
+        "xdm:contactID": {
```

Comment:
Same here. Remove the word "contact" from the property definitions as these will be under the nesting scope of xdm:contact